team-norms/bau_expectations.md

```diff
@@ -0,0 +1,20 @@
+# Business As Usual (BAU) Expectations
+
+Teams must deliver software systems in accordance with our Business As Usual Expectations that make software systems easier to run, debug, and improve.
+
+The Made Tech Support Team requirements are identified below. Where the application is transitioning to a customer support team, you should liaise with the customer team to understand their specific needs.
+
+## Application Requirements
+
+- Centralised Logging (Papertrail)
+- Application Performance Monitoring (Datadog)
+- Exception Tracking (Sentry)
+- Automated Dependency Upgrades (Dependabot)
```

Comment:
We're using package manager installed gitlab (k8s helm chart). 

Downside is that dependency upgrades don't work with gitlab. 

Upside is we have a make task that upgrades dependencies in < 30 seconds and tests that are safe enough to rely upon.

Is Dependabot a requirement?

Reply:
I'd say everything here is a requirement unless you can reach agreement with the support team that they're going to be able to support your own custom setup